pyuvsim/simsetup.py

```diff
@@ -879,19 +955,23 @@ def initialize_uvdata_from_params(obs_params):
     # Parse time structure
     time_dict = param_dict['time']
     uvparam_dict.update(parse_time_params(time_dict))
-    uvparam_dict['Npols'] = 4
 
-    # Now make a UVData object with these settings built in.
-    # The syntax below allows for other valid uvdata keywords to be passed
-    #  without explicitly setting them here.
+    # Parse polarizations
+    if 'polarization_array' not in uvparam_dict:
+        uvparam_dict['polarization_array'] = np.array([-5, -6, -7, -8])
```

Comment:
you might want to call this defaulting out in the docstring. Otherwise you have to read the code to figure out what the default is.